tests/loaders_test.py

```diff
@@ -557,38 +541,19 @@ def test_load_yaml_multiple_imports():
     oxygen_path = os.path.join(loaders_test_cases_path, 'multiple_imports/oxygen.yaml')
     water_path = os.path.join(loaders_test_cases_path, 'multiple_imports/water.yaml')
 
-    water_yaml = rules_loader.load_yaml(water_path)
-    assert water_yaml == {
-        'name': 'water',
-        'rule_file': water_path,
-        'symbol': 'O',
-    }
-    assert sorted(rules_loader.import_rules.keys()) == [
-        oxygen_path,
-        water_path,
-    ]
-    assert rules_loader.import_rules[oxygen_path] == [
-        hydrogen_path,
-    ]
-    assert rules_loader.import_rules[water_path] == [
-        oxygen_path,
-    ]
+    # re-load the rule a couple times to ensure import_rules cache is updated correctly
+    for i in range(3):
 
-    # reload the rule
-    water_yaml = rules_loader.load_yaml(water_path)
-    assert water_yaml == {
-        'name': 'water',
-        'rule_file': water_path,
-        'symbol': 'O',
-    }
-    assert sorted(rules_loader.import_rules.keys()) == [
-        oxygen_path,
```

Comment:
The `oxygen.yaml` file had no imports, so this is fixed now:

https://github.com/jertel/elastalert2/blob/42424d02cfd431a3e401b1fca1482517f2dc8687/tests/loaders_test_cases/multiple_imports/oxygen.yaml